tests/positive/partitions/creation.go

```diff
@@ -262,7 +262,7 @@ func ResizeRoot() types.Test {
 					"number": 9,
 					"size": 13008896,
 					"typeGuid": "3884DD41-8582-4404-B9A8-E9B84F2DF50E",
-					"guid": "3ED3993F-0016-422B-B134-09FCBA6F66EF",
```

Comment:
missed typeguid

Reply:
This typeGuid is special. I did not replace any guids that match:

"coreos-resize":   "3884DD41-8582-4404-B9A8-E9B84F2DF50E",
"data":            "0FC63DAF-8483-4772-8E79-3D69D8477DE4",
"coreos-rootfs":   "5DFBF5F4-2848-4BAC-AA5E-0D9A20B745A6",
"bios":            "21686148-6449-6E6F-744E-656564454649",
"efi":             "C12A7328-F81F-11D2-BA4B-00A0C93EC93B",
"coreos-reserved": "C95DC21A-DF0E-4340-8D7B-26CBFA9A03E0"